lib/app.js

```diff
@@ -1,17 +1,28 @@
 const express = require('express');
 const cookieParser = require('cookie-parser');
+const cors = require('cors');
 
-const authenticate = require('./middleware/authenticate');
+//const authenticate = require('./middleware/authenticate');
 
 const app = express();
 
 // Built in middleware
 app.use(express.json());
 app.use(cookieParser());
+app.use(
+  cors({
+    origin: [
+      'http://localhost:5500',
+      'https://alchemy-shopping-front-end-demo.netlify.app',
+    ],
+    credentials: true,
+  })
+);
 
 // App routes
 
 app.use('/api/v1/users', require('./controllers/users'));
+app.use('/api/v1/items', require('./controllers/items'));
```

Comment:
since every route in the items controller will be using the authenticate middleware, you should include it here ```app.use('/api/v1/items', authenticate, require('./controllers/items'));```